packages/domain/src/use-cases/publications/ToggleProperty.ts

```diff
@@ -1,43 +1,41 @@
-import { PublicationId } from '../../entities';
-
-export type TogglePropertyRequest = {
-  publicationId: PublicationId;
+export type TogglePropertyRequest<I> = {
+  id: I;
```

Comment:
If you make the `id: string` you won't need `I` generic, which simplifies a lot of changes in this PR.

Reply:
done